### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by genevec**
@ryanttb and I updated the readme to include representations for directories, as they likely won't be standard for all users. Also added a test step to the cron process, and placeholders for feature documentation that we don't yet include.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,7 +23,18 @@ The Amber module consists of two components:
...
+This documentation contains the following representations:
+* **$WEBROOT** - We represent where the root of your homepage is as $WEBROOT. (For example, in our install our webroot was /usr/local/nginx/html)
+* **$NGINXCONFDIR** -We also represent the Nginx configuration directory as $NGINXCONFDIR. (For example, in our install our Nginx configuration directory was /usr/local/nginx)
+* **$BUILDDIR** -We represent the build directory as $BUILDDIR. (For example, in our install our build directory was /usr/local/src)
+* **$DATADIR** -We represent the data directory as $DATADIR. (For example, in our install our data directory was /var/lib)
+* **$LOGDIR** -We represent the log directory as $LOGDIR. (For example, in our install our log directory was /var/log)
+* **$WEBROLE** -We run all services (nginx, FastCGI, database file owner, & cron), as a single user. We represent this user as $WEBROLE. (For example, in our install our web role user was www-data)
 
+Note that the Amber module reserves the /amber top-level directory. If you already have a top-level directory called Amber, you must modify the instructions.
+
```
Comment (jlicht) [inline README.md]: Might want to clarify that this refers to the top level directory from the perspective of the web server URLs, not the top level directory on the server.

File: README.md [hunk 2]
```diff
@@ -126,11 +145,23 @@ Insert Javascript and CSS required for Amber to function. `Required`
 
     subs_filter '</head>' '<script type="text/javascript" src="/amber/js/amber.js"></script><link rel="stylesheet" type="text/css" href="/amber/css/amber.css"></head>';
 
-Display Farsi version of Javascript and CSS 
+Display Javascript and CSS in Farsi
```
Comment (jlicht) [inline README.md]: This block can be removed, since it's been moved to "Optional Configuration" at the bottom of the page

**Comment by jlicht** [comment]
I've reviewed and added my (minor) comments.

File: README.md (hunk 1 above)
Comment (genevec) [inline README.md]: Fixed. In next pull request.

**Pull request closed without merging**